static/js/settings_config.ts

```diff
@@ -34,6 +34,21 @@ export const demote_inactive_streams_values = {
     },
 };
 
+export const right_sidebar_style_values = {
+    compact: {
+        code: 1,
+        description: $t({defaultMessage: "Compact"}),
+    },
+    with_status: {
+        code: 2,
+        description: $t({defaultMessage: "With status"}),
+    },
+    with_avatar: {
+        code: 3,
+        description: $t({defaultMessage: "With avatar"}),
+    },
```

Comment:
The `with_avatar` option should be commented out here, because we haven't implemented it yet.